auto-dark.el

```diff
@@ -49,13 +49,34 @@ if the themes are aware of `frame-background-mode', which many aren’t.
 If your themes aren’t aware of `frame-background-mode' (or you just prefer
 different themes for dark and light modes), you can set explicit lists of themes
 for each mode. Like with `custom-enabled-themes', the earlier themes in the list
-have higher precedence."
+have higher precedence.
+
+One other thing to be aware of is that when you first load a theme, you may be
+prompted to acknowledge that the theme can run arbitrary Lisp code.
+Acknowledging this and then allowing Emacs to treat the theme as safe in future
+sessions will silence the prompt (for that particular theme). If you would just
+prefer to ignore this warning for all themes, you can set ‘custom-safe-themes’
+to t."
   :group 'auto-dark
   :type '(choice
           (const :tag "Use custom-enabled-themes" nil)
           (list :tag "Use distinct dark & light lists"
                 (repeat :tag "Dark" symbol)
-                (repeat :tag "Light" symbol))))
+                (repeat :tag "Light" symbol)))
+  :set (lambda (_symbol value)
+         ;; Pre-load any themes used by Auto Dark (to force prompts for
+         ;; ‘custom-safe-themes’ while the user is interacting with Auto Dark,
+         ;; rather than at initialization or ‘frame-background-mode’ charge).
+         (mapc (lambda (mode-themes)
+                 (mapc (lambda (theme)
+                         (unless (custom-theme-p theme)
+                           (load-theme theme nil t)))
+                       mode-themes))
+               value)
+         ;; Make sure Auto Dark is showing the updated themes for the current
+         ;; ‘frame-background-mode’.
+         (auto-dark--check-and-set-dark-mode))
+  :version "0.13")
```

Comment:
Since we are changing stuff, I think now is the moment we should adopt SEMVER (https://semver.org/).

Would you please change the version on the top of the document to `0.13.1` and other occurrences of `0.13`? After this is on master, I'll manage to tag the `v0.13.1` and see if it solves #68 .

Also, you've done more than enough work to have your name after line 3. Feel free to add your name to the Authors list :smiley:

Reply:
Well thanks, I really appreciate that.

Re: the versioning. Glad to move to SemVer. I bumped the version, but left the other instances of "0.13" alone, since they refer to the earlier commit in which those things changed (including the one I added in this PR, since the variable was actually added before). _However_, I can bump them if the intent is to show that 0.13.1 is the first _published_ version they’re in.

Also, there’s no issue with this yet, but I made some notes about how to have versions that are compatible between Emacs’ internal versioning and SemVer: https://github.com/sellout/emacs-semver?tab=readme-ov-file#overlap-tldr, for future reference.

And finally, I think this PR stands on its own, but is in response to #67, so up to you whether it waits until that discussion is resolved.